common.mk

```diff
@@ -157,6 +157,30 @@ run-binary-debug: $(output_dir) $(sim_debug)
 
 run-fast: run-asm-tests-fast run-bmark-tests-fast
 
+#########################################################################################
+# helper rules to run simulator with fast loadmem via hex files
+#########################################################################################
+$(binary_hex): $(output_dir) $(BINARY)
+	hex_bytes=$(shell readelf --segments --wide $(BINARY) | grep LOAD | tail -n 1 | tr -s [:space:] | cut -f4,6 -d' ' | tr -d x | tr [:lower:] [:upper:] | tr ' ' + | sed 's/0000000008/ibase=16;/' | bc);\
```

Comment:
This is a pretty complicated series of commands. Since you're invoking python anyway, maybe it would be better to write a standalone python script and then call it here. There are some libraries you can use to read the ELF. If that's too much of a pain, you can just make it a bash script with the same commands.

Reply:
Do we have a place in chipyard to store python scripts?

Reply:
`./scripts`